docs/UserGuide.md

```diff
@@ -263,26 +322,52 @@ Format: `i-restock`
 
 #### 3.1 Adding an employee : `c-add`
 
-Adds an employee to the contact list.
+Adds an employee to the Employee Directory.
 
-Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG] ...`
+Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG]`
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+* In tCheck, you may add an employee to the Employee Directory. When you add an employee, you need to provide his/her name, phone number, emergency contact, and address.
+* Tag is optional.
 
-**Tip:**
-An employee can have any number of tags (including 0)
-</div>
+<div markdown="block" class="alert alert-info">
 
-Examples:
-* `c-add n/John Doe p/98765432 e/81234567 a/Blk 123 ABC Road`.
-* `c-add n/Betsy Crowe e/81234567 p/89007413 a/Blk 120 ABC Road t/Friday t/PartTime`.
+**:information_source: Note:** <br>
+PHONE_NUMBER and EMERGENCY_CONTACT must be 8-digit number and start with 8 or 9.<br>
+
+Example: 
+Let's say you want to add an employee called John Doe to the Employee Directory. His phone number is 98765432. His emergency contact is 81234567. His address is Blk 123 ABC Road. His tag is Friday and PartTime. You can follow these instructions:
+
+Steps to add an employee:
+1. Type `c-add n/John Doe p/98765432 e/81234567 a/Blk 123 ABC Road t/Friday t/PartTime` into the _Command Box_.
+1.  Press "Enter" to execute.
+
+Outcome:
+1. The _Result Display_ will show that you add a new employee.
+
+2. All active employees will be listed in the _Main View_.
 
-#### 3.2 Listing all employees : `c-list`
+![AddEmployeeCommandScreenshot](images/AddEmployeeResult.png)
 
-Shows a list of all employees in the contact list.
+#### 3.2 Listing all employees in the active list: `c-active-list`
 
+Lists all employees who are active.
+
+Format: `c-active-list`
+
```

Comment:
You may want to put an additional information section here to explain what does "active/unarchive" means.

Suggestions:

<div markdown="span" class="alert alert-primary">

**:information_source: Notes about Employee Directory:**<br>
* All employees who are currently working in the shop are active/unarchived employees. They are stored in the active(unarchived) Employee Directory. <br>
* All archived employees (refer to `c-archive` to see how to archive an employee) will be stored in the archived Employee Directory. You may refer to `c-archive-list` to view the archived Employee Directory.<br>
</div>